server/build/gzip.js

```diff
@@ -8,7 +8,8 @@ export default async function gzipAssets (dir) {
 
   const coreAssets = [
     path.join(nextDir, 'commons.js'),
-    path.join(nextDir, 'main.js')
+    path.join(nextDir, 'main.js'),
+    path.join(nextDir, 'static-bundle.js')
```

Comment:
@rauchg but the parsing time it so small compare to network time. So, it shouldn't be a much of a problem.

Anyway, I am not disagree for having just a single build file as that's how most of the other frameworks does. 

Shall we do this on a different issue/PR?